views/forms/ChartFilterForm/index.jsx

```diff
@@ -23,8 +25,21 @@ const ChartFilterForm = ({ initialValues, onSubmit }) => {
       },
       {}
     )
+    if (label === 'networks') {
```

Comment:
can we add constants for the string values like 'networks', 'sites', etc..

Reply:
f3c6984